ReSharper.FSharp/src/FSharp.Psi/src/FSharp.psi

```diff
@@ -1807,7 +1807,11 @@ exprStaticConstantTypeUsage extras {
   get { methodName="Expression" path=<exprStaticConstantTypeUsage:CHAMELEON_EXPR/chameleonExpression:EXPR> };
 }:
   CONST?
-  chameleonExpression<CHAMELEON_EXPR, ChameleonExpression>;
+  (
+    chameleonExpression<CHAMELEON_EXPR, ChameleonExpression>
```

Comment:
In what cases there's a chameleon and in what cases there's an expression?

Reply:
`A<"123">`
![image](https://user-images.githubusercontent.com/26364714/203619855-d2dba55c-db5d-4a89-8c3e-400cb31cecc1.png)

`A<const("123")>`
![image](https://user-images.githubusercontent.com/26364714/203619915-cbf77a71-0fa4-46ad-9c72-e1c65711fd6a.png)

Reply:
Why do we have different trees for these expressions? Looks like a tree builder bug to me

Reply:
Perhaps there were differences, because in the case of a literal in the compiler tree, only range is returned, not expr. 
It can be unified, but it seemed redundant to me to create a chameleon for a simple literal.

Reply:
I agree it seems redundant.

What I'm concerned about is in all other places we wrap expressions inside chameleons, even simple ones. We could stop doing it in some cases like this, but right now this case is just too inconsistent with the rest of the tree.

It helps when you have some simple/consistent rules like 'all top level expressions are wrapped in chameleons', but it turns out it's not the case, and now everyone has to take into account very rare cases where expressions aren't wrapped.